tutorials/backend/hasura-v3-ts-connector/tutorial-site/content/setup-connector.md

```diff
@@ -0,0 +1,607 @@
+{
+  "collection": "albums",
+  "query": {
+    "fields": {
+      "artist_id": {
+        "type": "column",
+        "column": "artist_id"
+      },
+      "id": {
+        "type": "column",
+        "column": "id"
+      },
+      "title": {
+        "type": "column",
+        "column": "title"
+      }
+    },
+    "limit": 10
+  },
+  "arguments": {},
+  "collection_relationships": {}
+}
+{"level":30,"time":1705491544622,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-3","res":{"statusCode":500},"responseTime":4.2100830078125,"msg":"request completed"}
+{"level":30,"time":1705491544623,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-4","req":{"method":"POST","url":"/query","hostname":"localhost:8100","remoteAddress":"127.0.0.1","remotePort":55462},"msg":"incoming request"}
+{
+  "collection": "albums",
+  "query": {
+    "aggregates": {
+      "count": {
+        "type": "star_count"
+      }
+    },
+    "limit": 10
+  },
+  "arguments": {},
+  "collection_relationships": {}
+}
+{"level":30,"time":1705491544624,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-4","res":{"statusCode":500},"responseTime":0.9287499785423279,"msg":"request completed"}
+{"level":30,"time":1705491544624,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-5","req":{"method":"POST","url":"/query","hostname":"localhost:8100","remoteAddress":"127.0.0.1","remotePort":55462},"msg":"incoming request"}
+{
+  "collection": "artists",
+  "query": {
+    "fields": {
+      "id": {
+        "type": "column",
+        "column": "id"
+      },
+      "name": {
+        "type": "column",
+        "column": "name"
+      }
+    },
+    "limit": 10
+  },
+  "arguments": {},
+  "collection_relationships": {}
+}
+{"level":30,"time":1705491544625,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-5","res":{"statusCode":500},"responseTime":0.482666015625,"msg":"request completed"}
+{"level":30,"time":1705491544625,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-6","req":{"method":"POST","url":"/query","hostname":"localhost:8100","remoteAddress":"127.0.0.1","remotePort":55462},"msg":"incoming request"}
+{
+  "collection": "artists",
+  "query": {
+    "aggregates": {
+      "count": {
+        "type": "star_count"
+      }
+    },
+    "limit": 10
+  },
+  "arguments": {},
+  "collection_relationships": {}
+}
+{"level":30,"time":1705491544625,"pid":47901,"hostname":"Seans-MBP.lan","reqId":"req-6","res":{"statusCode":500},"responseTime":0.37700000405311584,"msg":"request completed"}
+```
+
+In the logs of the app, we can see the request that was sent. It identifies the name of the collection, and a query 
+object to run. The query has a list of fields to retrieve, and a limit of 10 rows. With this as a guide, we can 
+start to implement our query function.
```

Comment:
Let's move this before the snippet so the user has some context before seeing the logs.